extensions/attention-stream/src/background.ts

```diff
@@ -54,6 +57,8 @@ const enableDevMode = Boolean(__ENABLE_DEVELOPER_MODE__);
 // eslint-disable-next-line no-undef
 const enableEmulatorMode = Boolean(__ENABLE_EMULATOR_MODE__);
 
+const GLEAN_MAX_URL_LENGTH = 2048;
```

Comment:
Where is this from, maybe glean could export a constant we could use?

Reply:
good point; filed an issue for this: https://github.com/mozilla/glean.js/issues/1508